apps/web/src/server/views/lpa/review-questionnaire.njk

```diff
@@ -0,0 +1,288 @@
+	{{ govukCheckboxes({
+		idPrefix: id + "-missing-or-incorrect",
+		name: id + "-missing-or-incorrect",
+		classes: "govuk-checkboxes--small",
+		formGroup: {
+			classes: 'govuk-!-margin-bottom-0'
+		},
+		errorMessage: completedError and {
+			text: completedError
+		},
+		items: [
+			{
+				value: "true",
+				text: "Missing or incorrect",
+				checked: completed === 'true',
+				conditional: conditionalHtml
+			}
+		]
+	}) }}
+{%- endmacro -%}
+
+{% block content %}
+	{{ super() }}
+
+	{% if errorSummary %}
+		{{ govukErrorSummary({
+			titleText: "There is a problem",
+			errorList: errorSummary
+		}) }}
+	{% endif %}
+
+	{{ govukBreadcrumbs({
+		classes: 'govuk-!-margin-top-0 govuk-!-margin-bottom-8',
+		items: [
+			{ href: backURL, text: 'Questionnaires for review' },
+			{ text: questionnaireData.AppealReference }
+		]
+	}) }}
+
+	<h1 class="govuk-heading-xl">
+		Review questionnaire
+	</h1>
+
+	<div class="govuk-grid-row">
+		<div class="govuk-grid-column-two-thirds">
+			{{ govukTable({
+				rows: [
+					[
+						{ html: '<strong>Appeal reference</strong>' },
+						{ text: questionnaireData.AppealReference }
+					],
+					[
+						{ html: '<strong>Appeal site</strong>' },
+						{ html: questionnaireData.AppealSiteHtml }
+					],
+					[
+						{ html: '<strong>Local planning department</strong>' },
+						{ text: questionnaireData.LocalPlanningDepartment }
+					]
+				]
+			}) }}
+		</div>
+	</div>
+
+	<form action="" method="post" novalidate>
+		<h2 class="govuk-heading-l govuk-!-margin-top-6">The planning application</h2>
+
+		{{ govukTable({
+			rows: [
+				[
+					{ html: '<strong>Planning application reference</strong>' },
+					{ text: questionnaireData.PlanningApplicationreference },
+					{ text: '' }
+				],
+				[
+					{ html: '<strong>Decision notice</strong>' },
+					{ html: '<a class="govuk-link" target="_blank" href="#">' + 'decision-notice.pdf' + '</a>' },
+					{ text: '' }
+				],
+				[
+					{ html: '<strong>Planning Officer\'s report</strong>' },
+					{ html: '<a class="govuk-link" target="_blank" href="#">' + 'planning-officers-report.pdf' + '</a>' },
+					{ html: missingOrIncorrectCheckbox(
+						'planning-officers-report',
+						fields.planningOfficersReport.completed,
+						fields.planningOfficersReport.error
+					)}
+				],
+				[
+					{ html: '<strong>Plans used to reach decision</strong>' },
+					{ html: '<a class="govuk-link" target="_blank" href="#">' + 'plans-used-to-reach-decision.pdf' + '</a>' },
+					{ html: missingOrIncorrectCheckbox(
+						'plans-used-to-reach-decision',
+						fields.plansUsedToReachDecision.completed,
+						fields.plansUsedToReachDecision.error,
+						{ html: detailsInput('textarea', 'plans-used-to-reach-decision', 'Which plans are missing?', fields.plansUsedToReachDecision.details)}
+					)}
+				]
+			]
+		}) }}
+
+		<h2 class="govuk-heading-l govuk-!-margin-top-9">Local plans and policies</h2>
+
+		{{ govukTable({
```

Comment:
This table is full of hardcoded documents right now... is that intentional?

Reply:
It was, the validation officer journey review-appeal template is the same currently. It seems as good a time as any to start adding this though, so I've added some code which renders these based on the documents data returned from the API (the data is still hardcoded in the API for now)

Reply:
The validation journey documents on the appeal details page are not hardcoded anymore.